asmcomp/power/emit.mlp

```diff
@@ -75,10 +75,6 @@ let (trap_size, trap_handler_offset, trap_previous_offset) =
   | ELF64v1 -> (32, 56, 64)
   | ELF64v2 -> (32, 40, 48)
 
-(* Output a symbol *)
-
-let emit_symbol s = Emitaux.emit_symbol '.' s
```

Comment:
If I understand correctly, this change (same for s390x below) changes the 'escaping character' from `.` to` $` on these architectures. Will that work, why was `.` not used before?

Reply:
Good, so this means that the whole escaping design is simpler than before, more consistent between backends.